modules/virt-creating-long-lived-account-and-token.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * virt/live_migration/virt-about-mtv-providers.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-creating-long-lived-account-and-token_{context}"]
+= Creating the long-lived service account and token to use with MTV providers
+
+(I need to add intro text here.)
+
+.Procedure
+
+[IMPORTANT]
+====
+QUESTION: Would adding the instructions/copying the sample file from the Early Access Test Plan be appropriate here? Alexander, you mention this in the PR's comments:
+We should explain how to create a Service Account (SA), and how to create a longer lived token associated with that SA. The hard part will be to define the RBAC needed for this Service Account. In our examples we use a cluster admin, but it may make sense to limit that.
```

Comment:
No the early access script will create a cluster admin user, which is what I am trying to avoid. That is way more privilege than is needed.

Reply:
Yes, I have gotten the RBAC from the MTV folks, just working on a procedure for you.